app/templates/jinja2/dashboard/volunteer/signup_players.html

```diff
@@ -49,6 +49,53 @@
       {% endfor %}
       </tbody>
     </table>
+    <table class="table table-striped">
+      <thead>
+      <tr>
+        <th scope="col">Self registration</th>
+        <th scope="col">Total registration</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td scope="col">{{ player_count - sum([loc.signupinvite_set.filter(used_at__isnull=False).count() for loc in locations]) }}</td>
+        <td scope="col">{{ player_count }}</td>
+      </tr>
+      </tbody>
+    </table>
+  {% endcall %}
+  {% call card("Signup Graph") %}
+  <div id="container" style="width: 75%;">
+    <canvas id="signup-graph"></canvas>
+  </div>
+
   {% endcall %}
 </div>
 {% endblock %}
+
+{% block scripts %}
+<script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.3/dist/Chart.min.js"></script>
+<script>
+
+    var config = {
+      type: 'line',
+      data: {
+        datasets: [{
+          data: {{ data|safe }},
```

Comment:
I'm unsure of the data structure that Charts.js expects. If possible, please test this locally before merging, that way we'll know if it actually is what you're expecting.